modules/openy_features/openy_block/modules/openy_block_branch_amenities/config/install/core.entity_form_display.paragraph.branch_amenities_with_icons.default.yml

```diff
@@ -0,0 +1,16 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.branch_amenities_with_icons.field_branch_amenties_with_icons
+    - paragraphs.paragraphs_type.branch_amenities_with_icons
+id: paragraph.branch_amenities_with_icons.default
+targetEntityType: paragraph
+bundle: branch_amenities_with_icons
```

Comment:
I can't find where this component being enabled during installation of OpenY
I bet this should go to Extended version of OpenY , somewhere in advanced editorial https://github.com/ymcatwincities/openy/blob/8.x-2.x/openy.packages.yml#L82

Reply:
Hi.
This is addition to existed feature.
Since, this block only added for Carnation, we just giving ability to use this paragraph but not adding it to demo content (bc at Rose at Lily it would not be styled).
During next phase we will style it for Rose and Lily and add this paragraph to the Demo content.